pkg/config/analysis/analyzers/testdata/conflicting-gateways-subSelector.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: networking.istio.io/v1
+kind: Gateway
+metadata:
+  name: beta
+spec:
+  selector:
+    istio: ingressgateway
+    app: istio-ingressgateway
+    cusKey: cusValue
+  servers:
+  - port:
+      number: 80
+      name: tcp
+      protocol: TCP
+    hosts:
+    - "foo.bar"
+---
+apiVersion: networking.istio.io/v1
+kind: Gateway
+metadata:
+  name: alpha
+spec:
+  selector:
+    istio: ingressgateway
```

Comment:
Does it not matter if `app: istio-ingressgateway` isn't included here in the selector?

Reply:
Hit the nail on the head

Reply:
we shoud judge based on the instances hit